R/custom_tidiers.R

```diff
@@ -248,10 +248,11 @@ pool_and_tidy_mice <- function(x, pool.args = NULL, ..., quiet = FALSE) {
 #' @rdname custom_tidiers
 #' @export
 tidy_gam <- function(x, conf.int = FALSE, exponentiate = FALSE, conf.level = 0.95, ...) {
-  broom::tidy(x,
```

Comment:
Not related to the swap. Is is intended?

Reply:
Yeah, this is related to the second issue listed in the PR (and yes, unrelated to the swap).